modules/Movecraft/src/main/java/net/countercraft/movecraft/craft/BaseCraft.java

```diff
@@ -368,9 +369,10 @@ public int getTickCooldown() {
             return type.getIntProperty(CraftType.SINK_RATE_TICKS);
 
         if (materials.isEmpty()) {
-            for (MovecraftLocation location : hitBox) {
-                materials.add(location.toBukkit(w).getBlock().getType());
-            }
+            //for (MovecraftLocation location : hitBox) {
+            //    materials.add(location.toBukkit(w).getBlock().getType());
+            //}
+            return 20;
```

Comment:
Why is this this way?

Reply:
This was a bit of debug code I left in, whoops!